File: src/tool/spring-boot-generator/output/authority/db/systemInfo.js
let systemInfo = {
	name: "systemInfo",
	description: "system info",
	columns: {
		id: {
			name: "id",
			description: "",
			isPk: true,
			autoIncrement: true,
			nullable: false,
			type: {
				name: "int",
				length: null
			}
		},
		name: {
			name: "name",
			description: "name",
			nullable: false,
			type: {
				name: "varchar",
				length: null
			}
		},
		description: {
			name: "description",
			description: "description",
			nullable: true,
			type: {
				name: "varchar",
				length: null
			}
		},
		createTime: {
			name: "createTime",
			description: "create time",
			nullable: false,
			type: {
				name: "timestamp",
				length: null
			}
		},
		createUser: {
			name: "createUser",
			description: "crceate user",
			nullable: false,
			type: {
				name: "varchar",
				length: null
			}
		}
	}
};

let columnsArray = [
	"id",
	"name",
	"description",
	"createTime",
	"createUser"
];

function create() {
  return {};
}

systemInfo.columnsArray=columnsArray;
systemInfo.create=create;
module.exports=systemInfo;

File: src/tool/spring-boot-generator/output/authority/authority/src/main/java/com/authority/pojo/entity/SystemInfo.java
package com.authority.pojo.entity;

import io.swagger.annotations.ApiModel;
import io.swagger.annotations.ApiModelProperty;
import lombok.Getter;
import lombok.Setter;

import java.util.Date;



@ApiModel("system info")
@Getter
@Setter
public class SystemInfo  {

    @ApiModelProperty("")
    private Integer id;

    @ApiModelProperty("name")
    private String name;

    @ApiModelProperty("description")
    private String description;

    @ApiModelProperty("create time")
    private Date createTime;

    @ApiModelProperty("crceate user")
    private String createUser;

}